### DISCUSSION TITLE
请问hysteria协议是否对优质线路比如CN2GIA-E 线路也有加速？

### DISCUSSION THREAD

**Discussion by QianHong** [apernet/hysteria#995] [General]
因为看到大家都说hysteria能够拯救垃圾线路，提升垃圾线路的网速。那么对于相对来说还可以的线路比如CN2 GIA-E 线路，是否也有加速效果呢？不知道有没有大佬验证过这个问题，有没有相关的结论？

**Comment by xiaorong61** [comment]
我觉得优质线路也有少量的波动，这时 hy2 就体现作用了，还有就是 hy2 自带无队头阻塞的多路复用（基于 tcp 的多路复用有队头阻塞问题，而基于 quic 的没有），可以降低握手延迟，以及相比 ss 更强的抗封锁能力。

**Comment by CodingXiaoheng2** [comment]
有的，与ss相比，能把美国CN2GIA的真连接延时降到ping延时的水平
